src/micromamba/config.cpp

```diff
@@ -184,6 +184,44 @@ set_config_describe_command(CLI::App* subcom)
     });
 }
 
+void
+set_config_get_command(CLI::App* subcom)
+{
+    auto& config = Configuration::instance();
+
+    auto& key = config.insert(Configurable("key", std::string())
```

Comment:
to disambiguate you can prepend the `Configurable` and the associated CLI option with `config_get_` for example

Reply:
osome
I see
Thank you! :cherry_blossom: